### PR TITLE
Create a struct to store configuration and ignore old events.

### PR DESCRIPTION
**Pull request by nikhitashanker77**
- Ignore old events.
- Create a struct that store configuration including workflow file, schedule file, etc to ensure that when the configuration changes, this does not cause tests to fail.

### CODE REVIEW

File: configs/config_factory.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package configs
```
Comment (MichaelBehm) [inline configs/config_factory.go]: Let's move all this into the configs.go file

File: workflow_scheduler_test.go [hunk 2]
```diff
@@ -203,12 +221,39 @@ func TestSubscriptionCallback_QueryWithCondition_2(t *testing.T) {
 	time.Sleep(5 * time.Second)
 }
 
+func TestSubscriptionCallback_OldEvent(t *testing.T) {
+	// Case 4: Call the SubscriptionCallback method with an event older than MaxAgeToProcessWinEvtsInDays in the config.
+	// Requires MaxAgeToProcessWinEvtsInDays in the test config is 1.
+
+	// Assume
+	ctrl := gomock.NewController(t)
```
Comment (MichaelBehm) [inline workflow_scheduler_test.go:229]: Explain how if the mockWorkflows functions get called, the Controller will error, failing this test

**Review by MichaelBehm — APPROVED** [review]
See my comments

**Comment by nikhitashanker77** [comment]
Resolved comments from code review.

**Pull request merged**